app/models/concerns/orchestration.rb

```diff
@@ -22,6 +22,14 @@ def rebuild_methods=(methods)
       @rebuild_methods = methods || {}
     end
 
+    def rebuild_methods_for(only = nil)
+      if only.present?
+        (@rebuild_methods || {}).select { |k,v| only.include?(v) }
+      else
+        @rebuild_methods || {}
+      end
+    end
```

Comment:
Setting only as defaulting to valid_rebuild_only_values breaks the tests that verified passing empty array would return all rebuild methods.  The use of `only.present?` was used to only filter rebuild methods when there was actual methods listed, so `nil` or `[]` would return all methods.  Should I modify this behavior so that an empty array returns no methods?